Lib/typing.py

```diff
@@ -2844,17 +2873,25 @@ class Point2D(TypedDict):
         Point2D = TypedDict('Point2D', {'x': int, 'y': int, 'label': str})
 
     By default, all keys must be present in a TypedDict. It is possible
-    to override this by specifying totality.
-    Usage::
+    to override this by specifying totality::
 
         class point2D(TypedDict, total=False):
             x: int
             y: int
 
-    This means that a point2D TypedDict can have any of the keys omitted.A type
+    This means that a point2D TypedDict can have any of the keys omitted. A type
     checker is only expected to support a literal False or True as the value of
     the total argument. True is the default, and makes all items defined in the
     class body be required.
+
+    The Required and NotRequired special forms can also be used to mark
+    individual keys as being required or not required::
+
+        class point2D(TypedDict):
```

Comment:
Lowercase looks weird here and is inconsistent with most other examples. Could you change this and the example immediately above to "Point2D"?